platform/ios/BUILD.bazel

```diff
@@ -213,7 +213,7 @@ genrule(
 xcodeproj(
     name = "xcodeproj",
     bazel_env = {"PATH": "/bin:/usr/bin:/usr/local/bin:/opt/homebrew/bin"},
-    build_mode = "xcode",
+    build_mode = "bazel",
```

Comment:
A couple things:
- bwb is really slow, it isn't building in parallel. There might be a way to fix that though.
- bwb fails to locate a provisioning profile which is why we switched to bwx. I tested building the demo app and it still fails here in bwb mode.

Reply:
It does find the provisioning profile for me now. Could we make a bug report over at https://github.com/MobileNativeFoundation/rules_xcodeproj ?

BwX does not have first-class support from rules_xcodeproj and support for it might be removed altogether in the future. I will add your comment to the discussion [here](https://github.com/MobileNativeFoundation/rules_xcodeproj/discussions/2391).

Reply:
I made the `BUILD_MODE` configurable. You can set it to your liking in your `config.bzl`.